newdle/client/src/components/creation/CreationHeader.js

```diff
@@ -73,8 +74,8 @@ Step.defaultProps = {
 function StepBody({title, description}) {
   return (
     <div className={styles['step-body']}>
-      <h3>{title}</h3>
-      <div className={styles.description}>{description}</div>
+      <h3>{t`${title}`}</h3>
+      <div className={styles.description}>{t`${description}`}</div>
```

Comment:
There's nothing to translate here and by wrapping a variable with the translation marker the actual strings won't get extracted. So you need to add the `t` where the `steps` are defined instead.

Reply:
Got it, thanks. Still having build issues so it's hard to test. Hopefully Mac pushes an OS fix so I can contribute more later on. Something with `cffi` is erroring at the moment.

Reply:
Don't worry, I'll test it. Do you manage to run `make i18n` for the i18n extraction? Otherwise I can push the update to the po file before merging.

Reply:
Would you mind pushing the update? That is broken too on my end. Thanks

Reply:
there we go :)